easybuild/tools/filetools.py

```diff
@@ -1455,3 +1457,29 @@ def copy_dir(path, target_path, force_in_dry_run=False):
             _log.info("%s copied to %s", path, target_path)
         except (IOError, OSError) as err:
             raise EasyBuildError("Failed to copy directory %s to %s: %s", path, target_path, err)
+
+
+def copy(paths, target_path, force_in_dry_run=False):
+    """
+    Copy single file/directory or list of files and directories to specified location
+
+    :param paths: path(s) to copy
+    :param target_path: target location
+    :param force_in_dry_run: force running the command during dry run
+    """
+    if isinstance(paths, basestring):
+        paths = [paths]
+
+    _log.info("Copying %d files & directories to %s", len(paths), target_path)
+
+    for path in paths:
+
+        full_target_path = os.path.join(target_path, os.path.basename(path))
```

Comment:
No, `target_path` is the directory we're copying to, `full_target_path` is the path that also includes the name of the file/directory we're copying to...